### PR TITLE
Fix admin UI, kill logged actions, and fix NPE in IlluminaNotificationDto

### PR DESCRIPTION
**Pull request by apmasell**

### CODE REVIEW

**Comment by sqwci** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [IlluminaNotificationDto.java#L80](https://github.com/TGAC/miso-lims/blob/b3450e8ca2c35f894735a4cd23b40a927ee50403/miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/IlluminaNotificationDto.java#L80): Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'

File: miso-web/src/main/webapp/scripts/admin_ajax.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                        Utils
+                            .showOkDialog(
+                                'Cache',
+                                results
+                                    .filter(function(result) {
+                                      return result.count > 0;
+                                    })
+                                    .map(
+                                        function(result) {
+                                          return "Regenerated " + result.count + " barcodes for " + result.target + ".";
```
Comment (djcooke) [inline miso-web/src/main/webapp/scripts/admin_ajax.js]: Controller logs and ignores errors, and then this only shows the success count. Unless you know the exact number of objects you expect this to work for (or check the log when you have no reason to), there's no indication that anything failed. Should at least count failures and notify the user

Comment (apmasell) [reply miso-web/src/main/webapp/scripts/admin_ajax.js]: Okay. Included some more stats.

**Comment by sqwci** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [IlluminaNotificationDto.java#L80](https://github.com/TGAC/miso-lims/blob/a9b6393d4d443e432a8f09d3714f4555ef696eb0/miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/IlluminaNotificationDto.java#L80): Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'

**Pull request merged**